src/main/resources/application.properties

```diff
@@ -1,14 +1,13 @@
-
-server.port=8081
+server.port=${SERVER_PORT:8081}
 
 # DB
 spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.PostgreSQLDialect
 spring.jpa.hibernate.ddl-auto=update
 spring.jpa.hibernate.show-sql=true
-spring.datasource.url=jdbc:postgresql://localhost:5432/url_shortner
-spring.datasource.username=postgres
-spring.datasource.password=postgres
+spring.datasource.url=jdbc:postgresql://${HOST}:${POSTGRES_PORT}/${POSTGRES_DB_NAME}
+spring.datasource.username=${POSTGRES_USERNAME}
+spring.datasource.password=${POSTGRES_PASSWORD}
 
 # Security
-spring.security.oauth2.resourceserver.jwt.issuer-uri=http://localhost:8080/auth/realms/login-app
-spring.security.oauth2.resourceserver.jwt.jwk-set-uri=http://localhost:8080/auth/realms/login-app/protocol/openid-connect/certs
+spring.security.oauth2.resourceserver.jwt.issuer-uri=${KEYCLOAK_REALM}
+spring.security.oauth2.resourceserver.jwt.jwk-set-uri=${JWT_TOKEN}
```

Comment:
You can use `DB_HOST`, `DB_USERNAME`, `DB_PASSWORD`, `DB_PORT`, `DB_NAME`
for keycloak `KEYCLOAK_AUTH_URL` & `KEYCLOAK_REALM`

eg. 

spring.security.oauth2.resourceserver.jwt.issuer-uri=`${KEYCLOAK_AUTH_URL}`/realms/`${KEYCLOAK_REALM}`

Reply:
ok